tests/test_controller.py

```diff
@@ -118,7 +118,7 @@ def close_windows(self):
             # HACK: make sure app is really closed
             time.sleep(0.5)
 
-    def wait_end(self, subprocess_pipe, timeout=30):
+    def wait_end(self, subprocess_pipe: bool, timeout: int = 30) -> bool:
```

Comment:
Let's simply replace `subprocess_pipe: Any` and return `int` here

Reply:
Done